runelite-client/src/main/java/net/runelite/client/plugins/itemstats/ItemStatChanges.java

```diff
@@ -228,11 +289,16 @@ private void init()
 		add(combo(2, heal(HITPOINTS, 11), boost(AGILITY, 5), heal(RUN_ENERGY, 10)), SUMMER_PIE, HALF_A_SUMMER_PIE);
 		add(combo(2, heal(HITPOINTS, 10), boost(FLETCHING, 4)), DRAGONFRUIT_PIE, HALF_A_DRAGONFRUIT_PIE);
 
+		// Teas
+		add(combo(food(3), boost(ATTACK, perc(.02, 2))), CUP_OF_TEA_1978 /* Standard tea */);
```

Comment:
The attack boost is a static 2 boost according to the wiki, but I can see ingame that this is not accurate from testing ingame and getting a +3 boost. Do you have confirmation that 2% + 2 is the formula?

Reply:
It's +2 at level 1 and in the 40s, and +3 at level 99 and in the 50s. I don't remember the exact levels that were tested. That would be the only reasonable formula for those boosts.

Reply:
Looks like this was removed, perhaps in favor of the entry on L104? I think `CUP_OF_TEA_1978` can be added to it, and the attack boost changed to 2 + 2%.

Reply:
`CUP_OF_TEA` (id 712) is an unobtainable item. I changed it to `CUP_OF_TEA_1978` which is the obtainable version.